### PR TITLE
[IMP] website_sale: improve confirmation info

### PR DESCRIPTION
**Pull request by Yenthe666**
The sentence was a bit odd before this commit. After this commit it reads easier and it is more logical.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Yenthe666** [comment]
@mart-e or @dbo-odoo small improvement :)

File: addons/website_sale/static/src/js/website_sale_validate.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ publicWidget.registry.websiteSaleValidate = publicWidget.Widget.extend({
                 } else {
                     var $message = $(result.message);
                     var $warning =  $("<i class='fa fa-warning' style='margin-right:10px;'>");
-                    $warning.attr("title", _t("We are waiting the confirmation of the bank or payment provider"));
+                    $warning.attr("title", _t("We are waiting for confirmation of the bank or the payment provider"));
```
Comment (dbo-odoo) [inline addons/website_sale/static/src/js/website_sale_validate.js]:
```suggestion
                    $warning.attr("title", _t("We are waiting for confirmation from the bank or the payment provider"));
```

Then I suggest to modify this as well )

Comment (Yenthe666) [reply addons/website_sale/static/src/js/website_sale_validate.js]: I agree. I'll do a change later today =)

**Comment by Yenthe666** [comment]
@dbo-odoo done

**Comment by dbo-odoo** [comment]
@robodoo r+

**Comment by Yenthe666** [comment]
Thanks :)

**Comment by dbo-odoo** [comment]
today is merge day it would seem :p

thx for the conrtibution!

**Comment by Yenthe666** [comment]
Love merges haha! You're welcome, as always :)

**Pull request closed without merging**